marlowe-client/src/Control/Monad/Trans/Marlowe.hs

```diff
@@ -28,76 +28,17 @@ import Control.Monad.Writer (MonadWriter)
 import Data.GeneralAllocate (GeneralAllocate(..), GeneralAllocated(..))
```

Comment:
Changes:

- Removed MarloweTracedT because the connection refactor made it redundant.